package.json

```diff
@@ -48,7 +48,7 @@
     "statements": 17,
     "branches": 13,
     "functions": 25,
-    "lines": 18
+    "lines": 17
```

Comment:
Is that codecov or something?

Reply:
it shows in testing 17.99%, therefore I discussed with @nemoshlag to increase it.